modules/nav_functions.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/prefer-default-export */
+export const configNavItems = () => {
```

Comment:
- [ ] I suggest that you should not disable any `eslint` guidelines set for this project because `eslint` check clean code and javascript best practices usage :blush: I think `configNavItems` is the only function you are exporting in this file so eslint is recommending for you to use the `export default` :relaxed: